package.json

```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "@openzeppelin/contracts": "5.0.2",
-    "@prb/math": "4.0.3"
+    "@prb/math": "4.1.0"
```

Comment:
Should be changed in peerDependencies as well.

Reply:
correct, missed it
it should be: "4.x.x"

Reply:
Thanks. Should also be updated in README where we have installation guidelines.

Reply:
yes, we should